cpp/include/Ice/OutputStream.h

```diff
@@ -567,14 +567,14 @@ class ICE_API OutputStream : public IceInternal::Buffer
      * Writes a short to the stream.
      * @param v The short to write.
      */
```

Comment:
Same here, please update the doc comment. For example:
Marshals an int16_t as a Slice short.